tests/ttnn/unit_tests/test_graph_capture.py

```diff
@@ -35,3 +35,119 @@ def test_graph_capture(tmp_path, device, scalar, size, mode):
+    )
+    # ttnn::prim::permute
+    assert (
+        captured_graph[4]["arguments"]
+        == "[ Tensor(storage=DeviceStorage(memory_config=MemoryConfig(memory_layout=TensorMemoryLayout::INTERLEAVED,buffer_type=BufferType::L1,shard_spec=std::nullopt)),tensor_spec=TensorSpec(logical_shape=Shape([1, 2048, 4, 128]),tensor_layout=TensorLayout(dtype=BFLOAT16,page_config=PageConfig(config=RowMajorPageConfig(tile=Tile(tile_shape={32, 32},face_shape={16, 16},num_faces=4))),memory_config=MemoryConfig(memory_layout=TensorMemoryLayout::INTERLEAVED,buffer_type=BufferType::L1,shard_spec=std::nullopt),alignment=Alignment([1])))), std::__1::reference_wrapper<tt::tt_metal::Tensor const>],[ SmallVector([0, 2, 1, 3]), std::__1::reference_wrapper<tt::stl::SmallVector<unsigned int, 8ul>>],[ MemoryConfig(memory_layout=TensorMemoryLayout::INTERLEAVED,buffer_type=BufferType::L1,shard_spec=std::nullopt), std::__1::reference_wrapper<tt::tt_metal::MemoryConfig const>],[ unsupported type , std::__1::reference_wrapper<std::__1::nullopt_t const>],[ 0, std::__1::reference_wrapper<std::__1::optional<float> const>]"
+    )
+    # PermuteDeviceOperation
+    assert (
+        captured_graph[6]["arguments"]
+        == "[ unsupported type , std::__1::reference_wrapper<ttnn::operations::data_movement::PermuteDeviceOperation::operation_attributes_t const>],[ unsupported type , std::__1::reference_wrapper<ttnn::operations::data_movement::PermuteDeviceOperation::tensor_args_t const>]"
+    )
+    # tt::tt_metal::create_device_tensor
+    assert (
+        captured_graph[7]["arguments"]
+        == "[ Shape([1, 4, 2048, 128]), std::__1::reference_wrapper<tt::tt_metal::Shape const>],[ BFLOAT16, std::__1::reference_wrapper<tt::tt_metal::DataType>],[ Row Major, std::__1::reference_wrapper<tt::tt_metal::Layout>],[ unsupported type , std::__1::reference_wrapper<tt::tt_metal::v0::IDevice*>],[ MemoryConfig(memory_layout=TensorMemoryLayout::INTERLEAVED,buffer_type=BufferType::L1,shard_spec=std::nullopt), std::__1::reference_wrapper<tt::tt_metal::MemoryConfig const>]"
```

Comment:
wdyt if we remove the type specification when a value is captured?
```suggestion
        == "Shape([1, 4, 2048, 128]), BFLOAT16, Row Major,[ unsupported type , tt::tt_metal::v0::IDevice*, MemoryConfig(memory_layout=TensorMemoryLayout::INTERLEAVED,buffer_type=BufferType::L1,shard_spec=std::nullopt)"
```

Reply:
sure would be better to see `tt::tt_metal::Layout::ROW_MAJOR` instead of `Row Major`, but I'd start with just a value string if to_string for the type is defined, expecting that a nice to_string includes enough type info

Reply:
I actually did the type specification on purpose, so if you get an undefined or something that is not completely clear, you could figure out from the type, even add new types based on the log

Reply:
fixed!